docfx/doxygen2syntax.cc

```diff
@@ -378,8 +377,8 @@ void AppendFunctionSyntax(YAML::Emitter& yaml, YamlContext const& ctx,
       }
       yaml << YAML::EndMap;
     }
-    // Generate the template parameters as normal parameters, there does not
-    // seem to be other ways to define them.
+    // Generate the template parameters as normal parameters, as there does not
+    // seem to be other way to document them.
```

Comment:
Nit: `s/other way/any other way/` OR `s/other way/another way/`?

Reply:
missed that, will fix in the next PR.